osu.Game/Screens/OnlinePlay/Lounge/Components/RoomAccessType.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Screens.OnlinePlay.Lounge.Components
+{
+    public enum RoomAccessType
+    {
+        All,
+        Public,
+        Private
```

Comment:
I don't think locked rooms are necessarily private. When filtering "private", I expect to see rooms exclusive to me and not random locked rooms. You can say that for public and private repos, for example.

Apart from that, when playlists reach parity (i.e. get passwords) and implement all the greyed-out settings:

![image](https://user-images.githubusercontent.com/35318437/175356407-96986427-5ad8-4702-bf0b-c864d1d5aab8.png)

what will "room visibility" be?

Reply:
Can we not complicate matters more than they need to be complicated? "Private" does not mean "private to me only". It's only one interpretation of the word.

Also disabled buttons added years ago that haven't even been implemented yet should have no bearing on this filter.